src/components/NavBar/NavBar.css

```diff
@@ -29,12 +29,73 @@ img {
 }
 
 .ontoggle-open {
-  margin-left: 240px;
   filter: blur(3px) opacity(40%);
```

Comment:
Don't you I think it would be better for UX if the background didn't get blurred so the user could see where he was redirected?

Also when clicking in a link when the nav is shrinked, the nav bar shouldn't expand. Maybe the user already knows the icons so he/she doen't need to see more details on the NAV